.gitignore

```diff
@@ -22,12 +22,5 @@ parts/
 src/collective.cover/
 var/
 /dist/
-src/brasil/gov/temas/themes/padrao/brasilgovtemas.js
```

Comment:
this files are being generated on the wrong location:

```console
$ git status
On branch add-hash-static-resources
Your branch is up-to-date with 'origin/add-hash-static-resources'.
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	modified:   .gitignore

Untracked files:
  (use "git add <file>..." to include in what will be committed)

	src/brasil/gov/temas/themes/brasilgovtemas.5ac944da71b7f3a87495.css
	src/brasil/gov/temas/themes/brasilgovtemas.5ac944da71b7f3a87495.css.map
	src/brasil/gov/temas/themes/brasilgovtemas.6432bdc6bee13e3efe4d.css
	src/brasil/gov/temas/themes/brasilgovtemas.6432bdc6bee13e3efe4d.css.map
	src/brasil/gov/temas/themes/brasilgovtemas.ddc6d9f01449c379de0c.css
	src/brasil/gov/temas/themes/brasilgovtemas.ddc6d9f01449c379de0c.css.map

no changes added to commit (use "git add" and/or "git commit -a")
```

Reply:
fixed in the last commit